walkthroughs/howto-k8s-appmesh-load-test/cluster.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: basic-cluster
+  region: us-west-2
+
+nodeGroups:
+  - name: ng-1
+    instanceType: m5.4xlarge
+    desiredCapacity: 10
```

Comment:
Is such a big group intentional? I usually see at most 5 in other examples.

Reply:
It is just an example cluster config showing customer how to use a cluster config as a file to create cluster with various properties such as different instance size.